src/wallet/wallet.h

```diff
@@ -888,6 +879,15 @@ class CWallet final : public CCryptoKeyStore, public CValidationInterface
      */
     const CTxOut& FindNonChangeParentOutput(const CTransaction& tx, int output) const;
 
+    /**
+     * Select a set of coins such that nValueRet >= nTargetValue and at least
+     * all coins from coinControl are selected; Never select unconfirmed coins
+     * if they are not ours
+     */
+    bool SelectCoins(const std::vector<COutput>& vAvailableCoins, const CAmount& nTargetValue, std::set<CInputCoin>& setCoinsRet, CAmount& nValueRet, const CCoinControl *coinControl = nullptr) const;
+
+    bool SelectAssets(const std::map<std::string, std::vector<COutput> >& mapAvailableAssets, const std::map<std::string, CAmount>& mapAssetTargetValue, std::set<CInputCoin>& setCoinsRet, std::map<std::string, CAmount>& nValueRet) const;
+
     /**
```

Comment:
Why this 200 line move? - I can't see any changes.

Reply:
Moved from private to public.

Reply:
github webpage camuflated that 👍